attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -53,6 +53,9 @@ public class AttributeMetadataModel implements Document {
   @JsonProperty(value = "display_name")
   private String displayName;
 
+  @JsonProperty(value = "description")
```

Comment:
nit (up to you): we can also move this definition after `private boolean internal;` so that it is the last addition in the object.

Reply:
Done.